### PR TITLE
Support Credential API

### PR DESCRIPTION
**Pull request by kvz**
Not ready yet, just trying to see if i can get basic interaction working.

Also see 

 - https://transloadit.slack.com/archives/C034DS4B1/p1657800061383759
 - https://github.com/transloadit/terraform-provider-transloadit/issues/32

### CODE REVIEW

**Comment by mifi** [comment]
Nice! should we implement a [live-api test](https://github.com/transloadit/node-sdk/blob/master/test/integration/__tests__/live-api.js) for this also?

**Comment by kvz** [comment]
@mifi The api is ready, and the example working, can you review this please?

File: examples/template_api.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ const template = {
 
 (async () => {
   try {
-    const { count } = await transloadit.listTemplates({ sort: 'created', order: 'asc' })
+    const res = await transloadit.listTemplates({ sort: 'created', order: 'asc' })
```
Comment (mifi) [inline examples/template_api.js:30]: what's the purpose of this change?

**Review by mifi — COMMENTED** [review]
would it be hard to implement a [live-api test](https://github.com/transloadit/node-sdk/blob/master/test/integration/__tests__/live-api.js) for this also?

File: examples/template_api.js (hunk 1 above)
Comment (kvz) [reply examples/template_api.js]: Reverted 👌

**Review by mifi — APPROVED** [review]
smooth!

**Comment by mifi** [comment]
had to revert your change from here: https://github.com/transloadit/node-sdk/pull/141/files#diff-7840a6781856ec257977e28eae3ace9ba049e3f214910ccff7d197e11ef992a6L640-R642

not sure if that change was intended, but `type` seems to be `undefined`, not `'s3'`.

**Comment by kvz** [comment]
Not intended no, apologies and thanks for cleaning up after me 👌
